src/templates/en/2019/methodology.html

```diff
@@ -3,12 +3,109 @@
 {% block title %}
 Methodology | The Web Almanac by HTTP Archive
 {% endblock %}
+ 
+{% block styles %}
+{{ super() }}
+<link rel="stylesheet" href="/static/css/methodology.css">
+{% endblock %}
 
 {% block main %}
-<h1>Methodology</h1>
-<p>TODO</p>
+<article id="methodology">
+  <nav class="index">
+    <div class="index-box floating-card">
+      <h2 class="header">Index</h2>
+      <h2 class="header-mobile">Index</h2>
+      <ul>
+        <li class="active">Lorem Ipsum Content
+          <ul>
+            <li>Lorem Ipsum Content
+              <ul>
+                <li>Lorem Ipsum Content</li>
+              </ul>
+            </li>
+          </ul>
+        </li>
+        <li>Lorem Ipsum Content</li>
+        <li>Lorem Ipsum Content</li>
+        <li>Lorem Ipsum Content</li>
+        <li>Lorem Ipsum Content</li>
+        <li>Lorem Ipsum Content</li>
+      </ul>
+    </div>
+  </nav>
 
-<h2>Overview of HTTP Archive</h2>
+  <section class="content">
+    <section class="intro">
+      <h1><div class="decorative-line"></div>Methodology<div class="decorative-line-mobile"></div></h1>
+      <p>A concise map into the guidelines that are strictly followed to output our reports.</p>
```

Comment:
I think this section is more of a subtitle than paragraph. The font appears similar to the title in the design:

![image](https://user-images.githubusercontent.com/1120896/66887066-b8c4e880-efa7-11e9-865c-40f8ad106f99.png)

For comparison:

![image](https://user-images.githubusercontent.com/1120896/66887044-a5198200-efa7-11e9-9b48-8d920fe07595.png)

I think the difference is that titles have the Poppins font while paragraphs use Lato.

Reply:
Oh, I overlooked it.
I will fix it!